Modules/Core/TestKernel/include/itkTestingMacros.h

```diff
@@ -67,15 +67,6 @@ namespace itk
 #define ITK_EXERCISE_BASIC_OBJECT_METHODS(object, ClassName, SuperclassName)                                           \
   object->Print(std::cout);                                                                                            \
   std::cout << "Name of Class = " << object->GetNameOfClass() << std::endl;                                            \
```

Comment:
@jhlegarreta I've added back the `std::cout` statement here. Is this sufficient for code coverage, i.e. can we get by without the explicit `std::strcmp` removed in the next line?

Reply:
@tbirdso should be OK. Thanks.